go/sdk/v1/state/rename.go

```diff
@@ -0,0 +1,7 @@
+package state
+
+func Rename(oldKey []byte, newKey []byte) {
```

Comment:
added a test that checks renaming from a non-existing key does not blow up

Reply:
Done.